nifi-nar-bundles/nifi-elasticsearch-bundle/nifi-elasticsearch-processors/src/main/java/org/apache/nifi/processors/elasticsearch/QueryElasticsearchHttp.java

```diff
@@ -462,7 +463,9 @@ private int getPage(final Response getResponse, final URL url, final ProcessCont
                 page.add(documentFlowFile);
             }
 
-            logger.debug("Elasticsearch retrieved " + responseJson.size() + " documents, routing to success");
+            if (logger.isDebugEnabled()) {
+                logger.debug("Elasticsearch retrieved " + responseJson.size() + " documents, routing to success");
```

Comment:
Should use a log formatting String instead, like other logger calls (rther than String concatenation)